README.md

```diff
@@ -107,6 +107,22 @@ Example request:
 }
 ```
 
+### Did Web Support
+Support for dids using the `did:web:domain` format can be configured using:
+
+```js
+import {config} from '@bedrock/core';
+import {Ed25519VerificationKey2018} from '@digitalbazaar/ed25519-verification-key-2018';
+
+config['vc-verifier'].methods.web = {
+  allowList: ['only-allow-this-domain:4644'],
+  fetchOptions: {size: 456000, agent: unsafeAgent},
+  verificationSuite: Ed25519VerificationKey2018
```

Comment:
Anything in the bedrock config will need to become default values for every verifier instance -- so we'll want to tweak the docs and perhaps rethink examples here. Specific configurations will need to be done via the verifier instance configs (and we don't need to add support for that right now), but devs should understand the bedrock config stuff will apply across all instances and should just set sane defaults.

Also, while the did web driver supports resolving to specific keys (and the above conflates a suite with a key type, so that should be fixed in the naming), that's really just a feature for people who are using old suites that haven't been properly updated to support multiple key types where appropriate. We probably shouldn't even make that a configurable option here in vc-verifier since it's somewhat of an anti-pattern generally. I guess that means that the name probably doesn't need to be fixed, we should just remove the option.

Anyway -- hopefully this comment is clear enough: many different verifier instances with different configs supporting different things will live on a given vc-verifier top-level app. The bedrock config only sets defaults that those will fall back to if instance config options (which we haven't defined yet and don't need to do right now) aren't set. So the bedrock config options need to be sane defaults that don't rule out use cases, i.e., you'd never set an allow list at that level.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/44/commits/269590aa3deb71432708a21a92f85f0194f88ff4

- Removed `allowList` from example
- Added warning about global.